### DISCUSSION TITLE
Send WhatsApp List message

### DISCUSSION THREAD

**Discussion by marcelotadeujr** [tulir/whatsmeow#669] [General]
Hi!
Someone here knows how send list message over whatsmeow and give some advide?

Thanks.

![image](https://github.com/user-attachments/assets/9d4a5bc7-ccc6-469c-bd1e-38b3bbaf19e5)

**Comment by hrizal** [comment]
you cannot do that from regular whatsapp.
use whatsapp business cloud API.

Pada Sel, 8 Okt 2024 pukul 20.10 Marcelo Tadeu ***@***.***>
menulis:

**Reply by marcelotadeujr** [reply]
Its not true, I did some research on source code and build that template by myself. I'ts worked for android, ios, web and business.
Thanks.

**Reply by KuntilBogel** [reply]
@marcelotadeujr Hey, would you like sharing the code, so we know, how to reproduce it using whatsmeow?

**Reply by YngviWarrior** [reply]
Any progress, i having the same problem... I did construct the ListMessage, there no errors, but is not showing on the destinatary conversation...
